app/Http/Controllers/CityController.php

```diff
@@ -0,0 +1,78 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use App\Models\City;
+use App\Models\Area;
+use Illuminate\Http\Request;
+
+class CityController extends Controller
```

Comment:
Add request field validation to all your endpoints
Use $request->validate function